### DISCUSSION TITLE
Some new programs to be added the script

### DISCUSSION THREAD

**Discussion by bash-ash** [ChrisTitusTech/winutil#998] [Ideas]
Hello? I have been using this script for a year, and its the best script for customizing/tweacking/debloating windows. Its super easy and convenient to use. But I have been missing some programs , that I often use in my daily basics. Those programs are just for tweackig and modifing the system

- BleachBit
- MemReduct
- VirtualBox

I think those apps would be useful to other people)

**Comment by anonyao** [comment]
Yeah bleachbit 100%

**Comment by FrostedVolcano** [comment]
Traffic monitor can also be added, it's a nice little software with very minimal resource usage that can show network, CPU and memory stats in the taskbar.
![image](https://github.com/ChrisTitusTech/winutil/assets/114462089/33b52700-4af3-492c-9400-1f0dfcad73e6)
![image](https://github.com/ChrisTitusTech/winutil/assets/114462089/e7f4e899-f548-487a-8ace-93a9d036327b)

**Comment by bash-ash** [comment]
And I also want to mention Notesnook - notetaking app , wich are not very know in the crowd, but its very good. Considering for review)

Terminal - Kitty, a Putty fork, that has new design, auto complete and GPU acceleration.
